test/test_nnapi.py

```diff
@@ -540,34 +540,29 @@ def test_conv2d(self):
                     )
 
     def test_conv2d_transpose(self):
+        torch.manual_seed(29)
         in_ch, out_ch, kernel = (5, 7, (2, 2))
         input_dim = (4, 5, 3, 3)
-        inp = torch.randn(input_dim)
         convert_dims = input_dim[:2] + (0, 0)
 
         for kind in ["float", "float-nhwc", "quant", "quant-nhwc"]:
             with self.subTest(kind):
+                inp = torch.randn(input_dim)
                 model = torch.nn.ConvTranspose2d(in_ch, out_ch, kernel)
                 output_size = model(inp).numel()
                 atol_rtol = (0.0002, 0)
                 limit = None
                 convert_arg = torch.zeros(*convert_dims)
 
                 if "quant" in kind:
-                    # FIXME 'aten::slow_conv_transpose2d' with arguments from the 'QuantizedCPU' backend
-                    continue
-                    model = torch.nn.Sequential(model)
-                    model.eval()
+                    model = torch.nn.quantized.ConvTranspose2d(in_ch, out_ch, kernel)
                     model.qconfig = torch.quantization.get_default_qconfig('qnnpack')
-                    model = torch.quantization.prepare(model)
-                    model(inp)
-                    model = torch.quantization.convert(model)
                     inp = qpt(inp, 1.0 / 16, 128)
                     # I've seen numerical differences between QNNPACK and NNAPI,
-                    # but never more than 1 quantum, and never more than ~1% of
+                    # but never more than 1 quantum, and never more than ~10% of
```

Comment:
Wait, is this for real?  9% deviation sounds like a bug

Reply:
Yeah did see up to 10% of elements being different up to 1 quantum, I can share a few examples

Reply:
Oh, wait.  I was reading this as 10% delta.  10% of 1 quantum, I guess that's life.